runtime/doc/deprecated.txt

```diff
@@ -29,7 +29,7 @@ HIGHLIGHTS
 
 LSP
 
-• todo
+• *vim.lsp.util.stylize_markdown()"
```

Comment:
For usages outside Neovim, should we suggest an alternative?

Reply:
Yeah, I don't know what though. @clason suggested we deprecate.

Reply:
`vim.treesitter.start()` 😇

Reply:
Could we mention the suggested alternative here?

Reply:
(and `cole=2`)